api/endpoint.go

```diff
@@ -159,19 +125,13 @@ func (e *Endpoint) fence(ctx *gin.Context, request FenceRequest) {
 	}
 
 	handle, err := vds.NewVDSHandle(conn)
+	if abortWithError(ctx, err) { return }
 	defer handle.Close()
-	if err != nil {
-		ctx.AbortWithError(http.StatusInternalServerError, err)
-		return
-	}
 
 	coordinateSystem, err := vds.GetCoordinateSystem(
 		strings.ToLower(request.CoordinateSystem),
 	)
-	if err != nil {
-		ctx.AbortWithError(http.StatusBadRequest, err)
-		return
-	}
+	if abortWithError(ctx, err) { return }
 
 	interpolation, err := vds.GetInterpolationMethod(request.Interpolation)
 	if err != nil {
```

Comment:
Plans to use `abortWithError` in other places, like here? Search for BadRequest still returns a couple of different places.